nov06/classWork/Project3of7.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
+   <title>My First JavaScript</title>
+   <script>
+	// *********************************************
+	// **    Code Author:   Jeff Kanakares
+	// **    Date: 10/13/2017
+	// **    Description: Welcome Script
+	// **
+	// **********************************************
+	// **       Change Log
+	// **     10/9/17 – Added header and changelog. Cliff Knopik
+	// **	  10/12/17 - Added welcome script. Jeff Kanakares
+	// ** .   10/22/17 - Added lesson 2 Jeff Kanakares
+	// **********************************************
+
+	// Put script here
+
+	// Prompt for name.
+	var person = prompt("Please enter your name");
+
+	//	If person enters name, display welcome message.
+	if (person != null) {
```

Comment:
This check will never work. The return from the prompt is an object. If someone presses `cancel` it will simply be an empty object. If someone presses `ok` without an entry, this will come back as an empty string. 

The code I wrote to ensure that this works follows:

```js
      // Loop userName prompt while input is true
      while (true) { // no need to specify all the types of entries

        // Declaring the variable userName and storing the user's name in it using prompt():
        const userName = prompt(`Welcome to Minesweeper. What is your name?`);

        // If userName var is set, then send response
        if (userName) {
          // Alert user of successful user name entry
          alert(`Hi, ${userName}. Good luck!`);
          break;
        }
      }
```